arangod/Agency/AsyncAgencyComm.h

```diff
@@ -90,8 +90,11 @@ struct AsyncAgencyCommResult {
       return Result{fuerteToArangoErrorCode(error), to_string(error)};
     } else {
       auto code = statusCode();
-      return Result{fuerteStatusToArangoErrorCode(code),
-                    fuerteStatusToArangoErrorMessage(code)};
+      auto internalCode = fuerteStatusToArangoErrorCode(code);
```

Comment:
This is a drive-by "fix".
If the FuerteResult was OK, and someone asked for the Result.
If was created with `TRI_ERROR_NO_ERROR` and message "200 OK".
The Result asserts that NO_ERROR has NO message and crashed.